backend/src/models/User.ts

```diff
@@ -4,6 +4,7 @@ import validator from 'validator';
 const isEmail = validator.isEmail;
 
 export interface IUser extends Document {
+  _id: mongoose.Types.ObjectId; 
```

Comment:
are you sure U ned to add it here? its a auto generated field

Reply:
yes, without this, TypeScript issues an error that doesn't accept the use of IUser in CRUD operations, mainly because there is a mismatch between types (and their respective attributes).